appgate/attrs.py

```diff
@@ -117,7 +178,41 @@ def _attrdump(d, value) -> Dict[str, Any]:
     dumper = datadumper.Dumper(**{})
     dumper.handlers.insert(0, (datadumper.is_attrs, _attrdump))
     dumper.handlers.insert(0, (is_datetime_dumper, lambda _a, v: dump_datetime(v)))
-    return dumper
+
+    if platform_type == PlatformType.K8S:
+        if api_spec is None:
+            raise AppgateTypedloadException(
+                "Unable to dump, APISpec is required",
+                platform_type=PlatformType.K8S,
+            )
+        else:
+            api = api_spec
```

Comment:
Necessary?

Reply:
Maybe a mypy workaround? I don't think its necessary either.

Reply:
It was a mypy workaround yes, I will try to remove it now since the code has changed

Reply:
```appgate/attrs.py:198: error: Argument 3 to "k8s_dumper" has incompatible type "Optional[APISpec]"; expected "APISpec"  [arg-type]
Found 1 error in 1 file (checked 30 source files)
make: *** [Makefile:12: lint] Error 1```